### PR TITLE
chore: circleci build

### PR DESCRIPTION
**Pull request by GutierrezGo**
References: https://autoricardo.atlassian.net/browse/CAR-9434

## Motivation
The second step for the branch deployment: circleci + upload.
The third step for the branch deployment: added the helm chart -> circleci deploys to k8s

## Take into consideration
I already added the webhook on gitHub.

# Thank you 🎄🐈

### CODE REVIEW

File: helm-chart/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    nginx.ingress.kubernetes.io/auth-secret: carforyou-components-pkg-basic-auth
+    nginx.ingress.kubernetes.io/auth-realm: "Authentication Required - carforyou"
+
+    # WARN: an invalid nginx directive may crash the ingress of the whole kubernetes cluster, please make triple sure your changes are valid
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      # whitelist office IPs (everything else uses basic-auth)
+      satisfy any;
+      allow 77.109.147.226; # bluecat_team (Altstetten)
+      allow 77.109.147.227; # bluecat_guest (Altstetten)
+      allow 178.83.153.152; # CarForYou (Stadelhofen)
```
Comment (lkappeler) [inline helm-chart/templates/ingress.yaml:20]: Not sure bout the Altstetten ones but we don't have the Stadelhofen office for a while so I think we can drop that  one

**Review by lkappeler — APPROVED** [review]
Looks good to me, deployment works fine good job!

File: helm-chart/templates/ingress.yaml (hunk 1 above)
Comment (GutierrezGo) [reply helm-chart/templates/ingress.yaml:20]: woooo good point! I will remove it from the header-footer as well.

Comment (GutierrezGo) [reply helm-chart/templates/ingress.yaml]: Maybe we can make another clean up in all the projects when we are fine about removing the ones of  `Altstetten`

**Comment by masone** [comment]
I'm so happy about this! can you change the link in the README here too please?

**Comment by GutierrezGo** [comment]
> I'm so happy about this! can you change the link in the README here too please?

https://github.com/carforyou/carforyou-components-pkg/pull/533 Done ✅

**Pull request merged**